app/src/main/java/org/mozilla/fenix/home/tips/TipManager.kt

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.tips
+
+import android.content.Context
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import org.mozilla.fenix.FeatureFlags
+
+enum class Type {
+    BUTTON
+}
+
+open class Tip(
+    @StringRes val identifier: Int,
+    @DrawableRes val icon: Int = -1,
+    @StringRes val title: Int,
+    @StringRes val description: Int,
```

Comment:
what are the thoughts behind making this resource values versus the actual string / drawables?

Reply:
Really happy with doing either here, do you think doing actual strings/drawables is cleaner? :)